CareKitStore/CareKitStoreTests/OCKStore/TestStore+BuildRevisions.swift

```diff
@@ -94,7 +94,7 @@ class TestStoreBuildRevisions: XCTestCase {
         try store.deleteTasksAndWait([taskA2])
         let revision = store.computeRevision(since: store.context.clockTime)
 
-        XCTAssert(revision.entities.count == 1)
+        XCTAssert(revision.entities.count == 2)
```

Comment:
This was revealed after fixing the logicalClock on a versioned outcome. It seems like it should be 2, as the added entity and the deleted entity should both be pushed to the remote

Reply:
Sounds right to me!